en/products/vpn/features.ftl

```diff
@@ -0,0 +1,87 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/features/
+
+vpn-features-page-title = Features - { -brand-name-mozilla-vpn }
+vpn-features-mozilla-vpn = { -brand-name-mozilla-vpn }
+
+# HTML page title
+# Line break for visual format only
+vpn-features-features-that-protect = Features that protect <br> your life online
+
+vpn-features-convenient = Convenient
+
+# Variables
+#   $servers (number) - number of VPN servers
+#   $countries (number) - number of available countries
+vpn-features-more-than = More than { $servers } servers in { $countries }+ countries
+vpn-features-see-our-list = See our list of servers.
+
+# Variables:
+#   $devices (number) - number of devices users can connect to VPN
+vpn-features-connect-up-to = Connect up to { $devices } devices
+vpn-features-supported-platforms = Supported on Windows, macOS, Android, iOS and Linux operating systems.
```

Comment:
```suggestion
vpn-features-supported-platforms = Supported on Windows, macOS, Android, iOS, and Linux operating systems.
```

Do we use Oxford comma in mozorg content? In case, there are a couple more to fix.

Reply:
I think the style guide says no Oxford comma.